app/models/financial_change_log.rb

```diff
@@ -0,0 +1,19 @@
+# FinancialChangeLog is an easy way to track all financial related data.
+#
+# While our app has already papertrail, the changing financial data through different CSV files or using different
+# data sources needs better transparency.
+# It provides one place where we can easy track:
+# * which process or ticket or dataset changed given data.
+# * which records and attributes of those records were changed
+# * if record is deleted, the log will still stay in database
+# * unlike per model versions, log can be easily searchable
+#
+# While NPQ is not fully a financial app, the FinancialChangeLog replicate to some extend a ledger functionality often being found in
+# proper financial apps.
+class FinancialChangeLog < ApplicationRecord
+  ONE_OFF_2326 = "OneOff 2326".freeze
+
```

Comment:
can we add some minimal validations, just check description and data changes are present - don't need to worry about error messages